### PR TITLE
added a deprecation flag to the wrangler init command

### PR DESCRIPTION
**Pull request by mxschumacher**
### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

File: content/workers/wrangler/commands.md [hunk 1]
```diff
@@ -153,6 +153,12 @@ wrangler docs [<COMMAND>]
 
 ## `init`
 
+{{<Aside type="note">}}
+
+This command has been deprecated and will be removed in a future version.
```
Comment (dario-piotrowicz) [inline content/workers/wrangler/commands.md]: Maybe we should provide the alternative? 🤔 

```suggestion
This command has been deprecated in favor of [C3](/pages/get-started/c3/) and will be removed in a future version.
```

(PS: it seem like we have a bit of a mess with the C3 links.... @kodster28 does the link below seem ok to you? shouldn't we have a single C3 page to always point to?)

**Review by dario-piotrowicz — APPROVED** [review]
Looks good to me, I just added a minor comment regarding the fact if the note should indicate the alternative 🙂 

Thanks @mxschumacher 😄

**Review by petebacondarwin — APPROVED** [review]
I agree with Dario.

**Comment by kodster28** [comment]
This still accurate given the convo we were having internally?

**Comment by KimJ15** [comment]
👋Hi there, we’re going to close this PR. All PRs prior to August 12 need to be reworked due to the platform migration to Astro.

**Pull request closed without merging**